packages/doc/docs/extensions/use-extension.mdx

```diff
@@ -0,0 +1,99 @@
+# Use Extension
+
+Besides built-in extensions, you could also use the published external extensions which built by VulcanSQL team or other contributor.
+
+## Install extension npm package
+
+VulcanSQL provides some external extension package for you could use it optionally, please see the [@vulcan-sql/](https://www.npmjs.com/search?q=%40vulcan-sql) prefix npm packages.
+
+:::info
+Not including `@vulcan-sql/core`, `@vulcan-sql/build` and `@vulcan-sql/serve` packages, this packages have installed globally when you installing `@vulcan-sql/cli`.
+:::
+
+## Declare used extensions
```

Comment:
Actually I',m not sure, because I consider setting `extensions` means that we use extensions, so use the `Declare used extensions` ( below code ):
```yaml
extensions:
  extension-name1: @vulcan-sql/ext-xxx
  extension-name2: contributor-ext
```
If this step is to install extensions, I will change it to `Declare installed extensions`.